### DISCUSSION TITLE
Additional Form Integration Guides

### DISCUSSION THREAD

**Discussion by stevenorechow** [webstudio-is/webstudio-community#80] [Wishlist]
Please consider adding guides for [Integrately](https://integrately.com/) and [ActivePieces](https://www.activepieces.com/) under [form integrations](https://github.com/webstudio-is/webstudio-community/blob/main/docs/university/marketplace/form-integrations/README.md).

**Comment by kof** [comment]
Hey @sorec574 would you like to contribute any of those guides following the other examples? You can send us a PR

**Comment by stevenorechow** [comment]
I think I can handle that, thank you!
